docs/resources/cluster.md

```diff
@@ -1,100 +1,334 @@
+    random = {
+      source  = "hashicorp/random"
+      version = "3.3.1"
+    }
+  }
+}
+
+resource "random_password" "password" {
+  length           = 16
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+variable "cluster_name" {
+  type        = string
+  description = "The name of the cluster."
+}
+
+
+resource "biganimal_cluster" "eha_cluster" {
+  cluster_name = var.cluster_name
+
+  allowed_ip_ranges {
+    cidr_block  = "127.0.0.1/32"
+    description = "localhost"
+  }
+
+  allowed_ip_ranges {
+    cidr_block  = "192.168.0.1/32"
+    description = "description!"
+  }
+
+  backup_retention_period = "6d"
+  cluster_architecture {
+    id    = "eha"
+    nodes = 5
+  }
+
+  instance_type = "aws:c5.large"
+  password      = resource.random_password.password.result
+  pg_config {
+    name  = "application_name"
+    value = "created through terraform"
+  }
+
+  pg_config {
+    name  = "array_nulls"
+    value = "off"
+  }
+
+  storage {
+    volume_type       = "gp3"
+    volume_properties = "gp3"
+    size              = "4 Gi"
+  }
+
+  pg_type            = "epas"
+  pg_version         = "14"
+  private_networking = false
+  cloud_provider     = "aws"
+  region             = "us-east-1"
+}
+
+output "password" {
+  sensitive = true
+  value     = resource.biganimal_cluster.eha_cluster.password
+}
+
+output "connection_uri" {
+  value = resource.biganimal_cluster.eha_cluster.connection_uri
+}
+```
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `cloud_provider` (String) Cloud Provider
-- `cluster_architecture` (Block List, Min: 1) Cluster Architecture (see [below for nested schema](#nestedblock--cluster_architecture))
+- `cloud_provider` (String) Cloud Provider. e.g. "aws" or "azure".
+- `cluster_architecture` (Block List, Min: 1) Cluster Architecture. See https://www.enterprisedb.com/docs/biganimal/latest/overview/02_high_availability/ for details. (see [below for nested schema](#nestedblock--cluster_architecture))
 - `cluster_name` (String) Name of the cluster.
-- `instance_type` (String) Instance Type
-- `password` (String, Sensitive) Password
-- `pg_type` (String) Postgres type
-- `pg_version` (String) Postgres Version
-- `region` (String) Region
-- `storage` (Block List, Min: 1) Storage (see [below for nested schema](#nestedblock--storage))
+- `instance_type` (String) Instance Type. e.g. "azure:Standard_D2s_v3", "aws:c5.large".
+- `password` (String, Sensitive) Password for the user edb_admin. It must be 12 characters or more.
+- `pg_type` (String) Postgres type. e.g. "epas", "pgextended" or "postgres".
+- `pg_version` (String) Postgres version. See https://www.enterprisedb.com/docs/biganimal/latest/overview/05_database_version_policy/#supported-postgres-types-and-versions for supported Postgres types and versions.
+- `region` (String) Region to deploy the cluster. See https://www.enterprisedb.com/docs/biganimal/latest/overview/03a_region_support/ for supported regions.
```

Comment:
```suggestion
- `region` (String) Region to deploy the cluster. See [Supported regions ](https://www.enterprisedb.com/docs/biganimal/latest/overview/03a_region_support/) for supported regions.
```

Reply:
Clickable link is added at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/0aea7392df32d1d3b7fcb6e35524aa585fa8a660